src/test/java/api/loans/anonymization/AnonymizeLoansByUserIdAPITests.java

```diff
@@ -0,0 +1,222 @@
+    UUID loanID = loanResource.getId();
+
+    sendCheckinLoanRequest(item1);
+
+    fakeAnonymizeLoansRequest(user.getId());
+
+    assertThat(loansStorageClient.getById(loanID)
+      .getJson(), isAnonymized());
+  }
+
+  @Test
+  public void canAnonymizeClosedLoansWithClosedFeesAndFines()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(
+        new CheckOutByBarcodeRequestBuilder().forItem(item1)
+      .to(user)
+      .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    accountsClient.create(new AccountBuilder().feeFineStatusOpen()
+      .withLoan(loanResource)
+      .feeFineStatusClosed()
+      .withRemainingFeeFine(150));
+
+    sendCheckinLoanRequest(item1);
+
+    fakeAnonymizeLoansRequest(user.getId());
+
+    assertThat(loansStorageClient.getById(loanID)
+      .getJson(), isAnonymized());
+  }
+
+  @Test
+  public void canAnonymizeMultipleClosedLoansWithClosedFeesAndFines()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    IndividualResource loanResource1 = loansFixture.checkOutByBarcode(
+        new CheckOutByBarcodeRequestBuilder().forItem(item1)
+      .to(user)
+      .at(servicePoint.getId()));
+
+    InventoryItemResource item2 = itemsFixture.basedUponNod();
+    IndividualResource loanResource2 = loansFixture.checkOutByBarcode(
+        new CheckOutByBarcodeRequestBuilder().forItem(item2)
+      .to(user)
+      .at(servicePoint.getId()));
+
+    UUID loanID1 = loanResource1.getId();
+    UUID loanID2 = loanResource2.getId();
+
+    accountsClient.create(new AccountBuilder().feeFineStatusOpen()
+      .withLoan(loanResource1)
+      .feeFineStatusOpen()
+      .withRemainingFeeFine(150));
+
+    sendCheckinLoanRequest(item1);
+    sendCheckinLoanRequest(item2);
+
+    fakeAnonymizeLoansRequest(user.getId());
+
+    assertThat(loansStorageClient.getById(loanID1)
+      .getJson(), not(isAnonymized()));
+
+    assertThat(loansStorageClient.getById(loanID2)
+      .getJson(), isAnonymized());
+  }
+
+  @Test
+  public void canNotAnonymizeLoansWithOpenFeesAndFines()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    IndividualResource loanResource = loansFixture.checkOutByBarcode
+        (new CheckOutByBarcodeRequestBuilder().forItem(item1)
+      .to(user)
+      .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    accountsClient.create(new AccountBuilder().feeFineStatusOpen()
+      .withLoan(loanResource)
+      .feeFineStatusOpen()
+      .withRemainingFeeFine(150));
+
+    sendCheckinLoanRequest(item1);
+
+    fakeAnonymizeLoansRequest(user.getId());
+
+    JsonObject json = loansStorageClient.getById(loanID).getJson();
+
+    assertThat(json, not(isAnonymized()));
+  }
+
+  private void sendCheckinLoanRequest(IndividualResource item)
```

Comment:
What value does this wrapper provide? As far as I can tell it only renames the activity, and in doing so includes technical language (e.g. send, request)

Reply:
Agree, removed